go/vt/vtgate/executor_select_test.go

```diff
@@ -596,7 +596,7 @@ func TestSelectKeyRangeUnique(t *testing.T) {
 func TestSelectIN(t *testing.T) {
 	executor, sbc1, sbc2, sbclookup := createExecutorEnv()
 
-	// Constant in IN is just a number, not a bind variable.
+	// Constant IN is just a number, not a bind variable.
```

Comment:
the edit is wrong. a clearer way to say this is `Constant in IN clause`.

Reply:
What do I need to change?